queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals;
+SELECT * from animals where name like '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-01-01';
+SELECT name from animals where escape_attempts <3;
+SELECT date_of_birth from animals where name = 'Agumon' or  name = 'Pikachu';
+SELECT name, escape_attempts from animals where weight_kg >10;
```

Comment:
- [ ] Let's be more consistent 🥲 `list name and escape attempts of animals that weigh more than 10.5kg` as you can see, it is 10.5 and not 10. Please correct this 👍🏻 

```sql
SELECT name, escape_attempts FROM animals WHERE weight_kg > '10.5';
```